pipeline/vars/bm_send_test_reports.groovy

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env groovy
+/*
+    Groovy script to publish the report generated by baremetal specific pipeline
+    Accepts test results <MAP> as parameter,
+    e.g :results = ["StageName1":[suite1: [Result: "Pass", Logs:"path to log"],
+                                  suite2: [Result: "PASS", Logs:"path to log"],
+                                  suite6: [Result: "PASS", Logs:"path to log"]],
+                    "StageName2":[suite3: [Result: "Pass", Logs:"path to log"],
+                                  suite4: [Result: "Fail", Logs:"path to log"],
+                                  suite5: [Result: "Fail", Logs:"path to log"]],
+                    "StageName3":[suite13: [Result: "Pass", Logs:"path to log"]]]
+*/
+
+def sendEMail(def test_results) {
+    /*
+        Send Email notification mentioning the stages, corresponding
+        suites and their execution status
+    */
+    def body = new File("pipeline/vars/emailable-report.html").text
```

Comment:
can you use **def body = readFile("pipeline/vars/emailable-report.html")** instead of **def body = new File("pipeline/vars/emailable-report.html").text**
This is throwing error when tried .
org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: Scripts not permitted to use new java.io.File java.lang.String

Reply:
Ack. Noted, the same shall me added